src/commands/edge_app.rs

```diff
@@ -614,27 +616,39 @@ impl EdgeAppCommand {
         Ok(())
     }
 
-    fn upload_edge_app_asset(
+    fn upload_edge_app_assets(
+        &self,
+        manifest: &EdgeAppManifest,
+        paths: &[PathBuf],
+    ) -> Result<(), CommandError> {
+        let pb = ProgressBar::new(paths.len() as u64);
+        pb.set_message("Files uploaded:");
+        let shared_pb = Arc::new(Mutex::new(pb));
+
+        paths.par_iter().try_for_each(|path| {
+            let result = self.upload_single_asset(manifest, path, &shared_pb);
```

Comment:
When it is 1 or 2 bigger files - progress bar will not be as granular as it was in sequential upload version. I guess we are ok with that though...

Reply:
yep